src/OpenTelemetry.Contrib.Instrumentation.EventCounters/InstrumentationType.cs

```diff
@@ -0,0 +1,52 @@
+﻿// <copyright file="InstrumentationType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Contrib.Instrumentation.EventCounters
+{
+    /// <summary>
+    /// Defines the type of the created instrumentation.
+    /// </summary>
+    public enum InstrumentationType
+    {
+        /// <summary>
+        /// Creates a counter instrumentation of long values which is an instrument that reports monotonically
+        /// increasing values.
+        /// </summary>
+        Counter = 0,
```

Comment:
Counter is an sync instrument, so we should not use it from EventCounter. 
Only supported instruments should be the ObservableCounter, ObservableGauge, and ObservableUpDownCounter (when that instrument is added)

Reply:
It's only named "Counter". The [publisher creates](https://github.com/hananiel/opentelemetry-dotnet-contrib/blob/b3a7d27320aaccf3a979b65f2fb445a60a29cd58/src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs#L86) an `ObservableCounter`.
Should we rename the `Counter` enum value to `ObservableCounter`?

Reply:
The names were renamed.